regression/goto-analyzer/simple-union-access/test.desc

```diff
@@ -0,0 +1,15 @@
+CORE
+union.c
+--variable-sensitivity --vsd --verify
```

Comment:
I think `--vsd` and `--variable-sensitivity` should do the same thing.  One was for backwards compatibility.

Reply:
Please can you go down to just one of these two?